apiron/endpoint/endpoint.py

```diff
@@ -154,3 +152,6 @@ def get_merged_params(self, supplied_params=None):
 
     def __str__(self):
         return self.path
+
+    def __repr__(self):
+        return "{klass}(path='{path}')".format(klass=self.__class__.__name__, path=self.path)
```

Comment:
```suggestion
        return "{class}(path='{path}')".format(class=self.__class__.__name__, path=self.path)
```

Reply:
I recommend we stick with `klass`.

Reply:
`class` is a reserved keyword. An alternative I've seen often is `class_`, but I find `klass` more readable somehow 😄